test/distributed/fsdp/test_fsdp_state_dict.py

```diff
@@ -482,6 +484,16 @@ def _dist_train(self, wrap_fsdp: bool, state_dict_type: str = ""):
             blank_model = FSDP(Model(True).cuda())
             _zero_model(blank_model)
             state_dict = self._state_dict(model, state_dict_type)
+            if move_to_cpu:
+                for key in list(state_dict.keys()):
+                    tensor = state_dict[key]
+                    if isinstance(tensor, torch.Tensor):
+                        state_dict[key] = tensor.cpu()
+                    else:
+                        shards = tensor.local_shards()
+                        if shards:
```

Comment:
Do we want an assertion here like `assert len(shards) == 1` to remind us that this test needs to change if we support multiple local shards?